src/DataAccess/DoctrineMembershipApplicationRepository.php

```diff
@@ -122,6 +124,16 @@ private function setPaymentFields( DoctrineApplication $application, MembershipP
 		$application->setPaymentIban( $bankData->getIban()->toString() );
 	}
 
+	private function setTrackingField( DoctrineApplication $application, MembershipApplicationTrackingInfo $trackingInfo ) {
+		if ( $trackingInfo->getKeyword() === '' ) {
+			$application->setTracking( '' );
+		} else {
+			$application->setTracking(
+				implode( '/', array_filter( [ $trackingInfo->getCampaignCode(), $trackingInfo->getKeyword() ] ) )
+			);
+		}
+	}
```

Comment:
Since this is just a single field, and $application->setTracking is always called, I'd have gone with `$application->setTracking( getTrackingField( $trackingInfo ) )`. Is slightly simpler